tests/Serializer/BaseSerializationTest.php

```diff
@@ -1356,6 +1356,10 @@ public function testTypedProperties()
             $this->markTestSkipped(sprintf('%s requires PHP 7.4', __METHOD__));
         }
 
+        $builder = SerializerBuilder::create($this->handlerRegistry, $this->dispatcher);
```

Comment:
@marein  Hi,
nice idea, and thanks for pull-request ! :+1: 

Are there any things to consider during Deserialization process, no changes there?

Would be possible to have a Deserialization test, maybe ? :)

Thanks!

Reply:
Hi @slava-v, thank you. The deserialization process is covered [here](https://github.com/schmittjoh/serializer/blob/8c596c628a0327eea84aa3d2724ed183dc681c0f/tests/Serializer/BaseSerializationTest.php#L1382). Did I miss something?